drf_to_mkdoc/utils/commons/auth_utils.py

```
@@ -0,0 +1,82 @@
+    
+    # Otherwise, treat as file path
+    auth_path = Path(auth_function_js)
+    
+    # If relative path, try relative to project root
+    if not auth_path.is_absolute():
+        # Try to find the project root (where manage.py or pyproject.toml is)
+        current_dir = Path.cwd()
+        project_root = None
+        
+        # Look for common project root indicators
+        for parent in [current_dir] + list(current_dir.parents):
```

Ruff (0.14.5)

Consider `[current_dir, *list(current_dir.parents)]` instead of concatenation

Replace with `[current_dir, *list(current_dir.parents)]`

(RUF005)